templates/emails/onboarding.day7.widgets.hbs

```diff
@@ -0,0 +1,21 @@
+Subject: Give visibility to your backers and sponsors
+
+{{> header}}
+
+<h1>Hey {{collective.name}} 👋</h1>
+
+<p>Did you know that we offer widgets to automatically show your backers and sponsors on your website? It's a great way to thank them for their financial contribution. It will also incentivize others to contribute as well!</p>
+
+<p>Go to the settings of your collective in the "export" tab: <a href="https://opencollective.com/{{collective.slug}}/edit#export">https://opencollective.com/{{collective.slug}}/edit#export</a></p>
+
+<p>And here are some examples: <a href="http://apex.run/#links">Apex</a>, <a href="https://www.spinacms.com">SpinaCMS</a>, <a href="https://webpack.js.org">Webpack</a> (using our API). You can also add a Donate Button or a widget to automatically show your events!</p>
+
```

Comment:
We are not using images here?

Reply:
Not yet. Cuiki will work on some and we will update then.

Reply:
ok